api/http/api_deployments.go

```diff
@@ -85,13 +85,23 @@ var (
 )
 
 type Config struct {
+	// URL signing parameters:
+
+	// PresignSecret holds the secret value used by the signature algorithm.
 	PresignSecret []byte
+	// PresignExpire duration until the link expires.
 	PresignExpire time.Duration
+	// PresignHostname fallback value for the signed url hostname.
+	PresignHostname string
+	// PresignScheme URL scheme to used for generating signed URLs.
+	PresignScheme string
 }
 
 func NewConfig() *Config {
 	return &Config{
-		PresignExpire: time.Second * 900,
+		PresignExpire:   time.Second * 900,
```

Comment:
I know, not part of the commit, but as long as you are here, maybe some explanation on the value of 900 and a const?

Reply:
It's just the default from the config file.

Reply:
LOL, I had created a constant without even using it :see_no_evil: